src/frontends/tensorflow/tests/unique_replacer.cpp

```diff
@@ -0,0 +1,133 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "helper_transforms/unique_replacer.hpp"
+
+#include <gtest/gtest.h>
+
+#include <frontend/shared/include/utils.hpp>
+#include <openvino/opsets/opset9.hpp>
+#include <openvino/pass/manager.hpp>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+#include "gtest/gtest.h"
+#include "helper_ops/unique.hpp"
+
+using namespace std;
+using namespace ov;
+using namespace opset9;
+using namespace element;
+using namespace frontend::tensorflow;
+using namespace frontend::tensorflow::pass;
+
+namespace {
+Output<Node> get_elements_number_1d(const Output<Node>& output, element::Type output_type) {
```

Comment:
You have same function in `utils.hpp` why you also add it here? Is it because of linking issue? Can we not have same functions in two places?

Reply:
That function is implemented for transformations and anticipate `NodeRegistry` so this will not work for the test.